src/domains/Townhall/TownHallHistory/TownHallHistory.tsx

```diff
@@ -0,0 +1,61 @@
+import React from 'react';
+
+import { createStyles, Theme, makeStyles } from '@material-ui/core/styles';
+import List from '@material-ui/core/List';
+import ListItem, { ListItemProps } from '@material-ui/core/ListItem';
+import ListItemIcon from '@material-ui/core/ListItemIcon';
+import ListItemText from '@material-ui/core/ListItemText';
+import Divider from '@material-ui/core/Divider';
+import InboxIcon from '@material-ui/icons/Inbox';
+import DraftsIcon from '@material-ui/icons/Drafts';
+import { ListItemAvatar, Avatar } from '@material-ui/core';
+import { getTownhallList } from '../api';
+
+const useStyles = makeStyles((theme: Theme) =>
+    createStyles({
+        root: {
+            width: '100%',
+            maxWidth: 360,
+            backgroundColor: theme.palette.background.paper,
+        },
+    })
+);
+
+
+
+function ListItemLink(props: ListItemProps<'a', { button?: true }>) {
+    return <ListItem button component='a' {...props} />;
+}
+
+export default function SimpleList(props: Props) {
+    const { townhall } = props;
+    const classes = useStyles();
+
+    return (
+        <div className={classes.root}>
+            <List component='nav'>
```

Comment:
I'm curious.  What's the purpose of changing this to use the `nav` tag?  I think it might make an accessibility issue arise if we have more than one `nav` tag present on the page.  This is sorta a note to myself to also double check.

Reply:
The nav tag is used in one of Material-UI List examples. It doesnt appear to violate the current accessibility scan, but removing it does not change intended functionality